tests/load_test/load_test.py

```diff
@@ -13,6 +13,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# added by Ashutosh Narayan
+
+import sys
+sys.path.append('../app/handlers/')
```

Comment:
If you add it here, it would only fix tests, but not the actual app. Can you follow:
https://github.com/google/personfinder/wiki/GettingStarted#step-3-run-the-app-locally
to run it locally and access it? I believe it currently fails with your change. There are two ways to fix it:

* Add something similar to app/main.py as well.
* Update `import` lines in all source files to include the subdirectory name e.g.,
    `import const` → `from utils import const`

I would prefer the second option which is more explicit. But that would involve more work. Would be appreciated if you follow the second option, but I'm OK if you follow the first option fo now, then someone will later switch to the second option.

Reply:
I have removed README.md file from handlers/ and also updated dictionary files.
Ran the tests locally after adding const module in all sources file but still tests failed. Can you review the new files with their changes which got committed and needs any further modification ? The tests are failing in import module itself both locally and during CI.

Reply:
@gimite Have fixed the unit tests and they are passing now, server_tests are failing though.